docusaurus.config.js

```diff
@@ -0,0 +1,137 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const USERNAME = 'Valayar';
+const REPO_NAME = 'EF_Informatik';
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Lern Blog',
+  tagline: 'EF Informatik',
+  url: `https://${USERNAME}.github.io/`,
+  baseUrl: `/${REPO_NAME}/`,
+  onBrokenLinks: 'warn',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: USERNAME, // Usually your GitHub org/user name.
+  projectName: REPO_NAME, // Usually your repo name.
+
+  // Even if you don't use internalization, you can use this field to set useful
+  // metadata like html lang. For example, if your site is Chinese, you may want
+  // to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'de',
+    locales: ['de'],
+  },
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          sidebarPath: require.resolve('./sidebars.js'),
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            `https://github.com/${USERNAME}/${REPO_NAME}/tree/main/`,
+        },
+        blog: {
+          routeBasePath: '/',
+          showReadingTime: true,
+          postsPerPage: 'ALL',
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            `https://github.com/${USERNAME}/${REPO_NAME}/tree/main/`,
+        },
+        theme: {
+          customCss: require.resolve('./src/css/custom.scss'),
+        },
+      }),
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        title: 'EF Blog',
+        logo: {
+          alt: 'Page Logo',
+          src: '/images/Logo_Website_Transparent_Black.png',
```

Comment:
Das Logo für die Webseite **muss** im Ordner `static/img` sein, sonst wird es von der Hauptseite nicht gefunden.

1. Logo dorthin verschieben
2. Pfad ändern zu
  ```suggestion
            src: 'img/Logo_Website_Transparent_Black.png',
  ```